hbase-common/src/main/java/org/apache/hadoop/hbase/HConstants.java

```diff
@@ -219,6 +222,12 @@ public enum OperationStatusCode {
   public static final String CLIENT_ZOOKEEPER_CLIENT_PORT =
     "hbase.client.zookeeper.property.clientPort";
 
+  /** Supported ZooKeeper client TLS properties */
+  public static final Set<String> ZOOKEEPER_CLIENT_TLS_PROPERTIES =
```

Comment:
Let's move this to ZKConfig directly? HConstants is IA.Public, but obviously we do not want our end users to use these things in their code?

Reply:
Done.